roles/mirror_ocp_release/tasks/artifacts.yml

```diff
@@ -90,9 +104,14 @@
       delay: 5
       until: _mor_install_mode is not failed
 
+    - name: Copy artifacts to release directory
+      ansible.builtin.copy:
+        src: "{{ _mor_tmp.path }}/"
+        dest: "{{ mor_cache_dir }}/{{ mor_version }}/"
```

Comment:
Please apply the SELinux context to the file in the release directory.

Reply:
Done.